content/en/docs/languages/cpp/getting-started.md

```diff
@@ -260,30 +268,21 @@ add_executable(dice-server ${SOURCES})
 
 set(OATPP_ROOT ../oatpp)
 set(OPENTELEMETRY_ROOT ../opentelemetry-cpp)
+set(opentelemetry-cpp_DIR ../otel-cpp/lib/cmake/opentelemetry-cpp)
 find_library(OATPP_LIB NAMES liboatpp.a HINTS "${OATPP_ROOT}/build/src/" NO_DEFAULT_PATH)
 if (NOT OATPP_LIB)
   message(SEND_ERROR "Did not find oatpp library ${OATPP_ROOT}/build/src")
 endif()
 # set the path to the directory containing "oatpp" package configuration files
 include_directories(${OATPP_ROOT}/src)
 
-include_directories(${OPENTELEMETRY_ROOT}/api/include)
-include_directories(${OPENTELEMETRY_ROOT}/sdk/include)
-include_directories(${OPENTELEMETRY_ROOT}/sdk/src)
-include_directories(${OPENTELEMETRY_ROOT}/exporters/ostream/include)
-
-find_library(OPENTELEMETRY_COMMON_LIB NAMES libopentelemetry_common.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/common" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_TRACE_LIB NAMES libopentelemetry_trace.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/trace" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_EXPORTER_LIB NAMES libopentelemetry_exporter_ostream_span.a HINTS "${OPENTELEMETRY_ROOT}/build/exporters/ostream" NO_DEFAULT_PATH)
-find_library(OPENTELEMETRY_RESOURCE_LIB NAMES libopentelemetry_resources.a HINTS "${OPENTELEMETRY_ROOT}/build/sdk/src/resource" NO_DEFAULT_PATH)
-
-if(OPENTELEMETRY_COMMON_LIB AND OPENTELEMETRY_TRACE_LIB AND OPENTELEMETRY_EXPORTER_LIB AND OPENTELEMETRY_RESOURCE_LIB)
-  message(STATUS "Found opentelemetry libraries")
-else()
-  message(SEND_ERROR "Did not find opentelemetry libraries")
-endif()
+# Use find_package to include OpenTelemetry C++
+find_package(opentelemetry-cpp CONFIG REQUIRED)
```

Comment:
can you verify that this is **not** loading packages from a global location (what happens if you add `NO_DEFAULT_PATH`)?

Reply:
When I add NO_DEFAULT_PATH I am telling CMake not to search in the default directories that it normally checks. It only uses/searches in this case:
set(opentelemetry-cpp_DIR ../otel-cpp/lib/cmake/opentelemetry-cpp)

Test:
```
$ grep find_package ../CMakeLists.txt 
# Use find_package to include OpenTelemetry C++
find_package(opentelemetry-cpp CONFIG REQUIRED NO_DEFAULT_PATH)

$ cmake ..
-- The C compiler identification is GNU 11.4.0
-- The CXX compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found opentelemetry-cpp: /home/ricardo/cncf/otel-cpp-starter/otel-cpp/include
-- Configuring done (0.4s)
-- Generating done (0.0s)
-- Build files have been written to: /home/ricardo/cncf/otel-cpp-starter/roll-dice/build

$ cmake --build .
[ 50%] Building CXX object CMakeFiles/dice-server.dir/main.cpp.o
[100%] Linking CXX executable dice-server
[100%] Built target dice-server
```

Reply:
can you add NO_DEFAULT_PATH then such that it is consistent with the oatpp config?

Reply:
done!